chart/values.yaml

```diff
@@ -562,6 +562,10 @@ workers:
       AND queue != '{{ .Values.config.celery_kubernetes_executor.kubernetes_queue }}'
       {{- end }}
 
+    # Weather to use PGBouncer to connect to the database or not when it is enabled
+    # This configuration will be ignored if PGBouncer is not enabled
+    usePgbouncer: true
```

Comment:
Should not we make it false by default? It changes the behaviour (of course) but likely the original behaviour was really a bug ?

Reply:
I agree it should be false by default, only because we don’t know how everyone’s postgresql is configured. It is possible, although unlikely, that connections from keda could be refused to the Postgres db due to pg_hba or networkpolicy, and this would be a breaking change on upgrade.

Reply:
Indeed, using false by default would potentially cause disruptions for certain users, particularly those who rely on an external database with restricted access.

Reply:
Fine for me. Good reason. I just wonder if we should have some more doc ("set it to true and make sure KEDA can directly access your DB if you see those logs....") ...

Not really needed though, sometimes having an issue or PR like that (indexed by search engines / chatbots) is enough of a visibility and we can always direct users to such discussions. So I am ok with merging as-is.

Reply:
I'm working on #10358 to improve KEDA doc, I will add some tips including this one.